src/app/shared/components/fy-userlist/fy-userlist-modal/fy-userlist-modal.component.ts

```
@@ -226,17 +212,13 @@ export class FyUserlistModalComponent implements OnInit, AfterViewInit {
               const searchTextLowerCase = searchText.toLowerCase();
               const newItem = {
                 isNew: true,
-                us_email: searchText,
+                email: searchText,
               };
-              const newArr = [];
+              const newArr: (Partial<Employee> & { isNew?: boolean })[] = [];
               newArr.push(newItem);
               newlyAddedItems = newArr.concat(newlyAddedItems);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)